commit/merkleroot/rmn/controller.go

```diff
@@ -660,9 +658,7 @@ func (c *controller) validateSignedObservationResponse(
 		}
 
 		// todo: The original updateReq contains abi encoded onRamp address, the one in the RMN response
-		// is 20 bytes evm address. This is chain specific and should be handled in a chain specific way.
```

Comment:
This whole todo seems suspect to me.
>> "he original updateReq contains abi encoded onRamp address"

@winder : Could you check this todo and code? It was wrong to clip the address to 20 bytes, but I didn't understand it completely.

I think @huangzhen1997's change is good, but want CCIP to review this closely.

Reply:
We need to keep this to pass RMN test. This should be addressed and removed